src/McBopomofo.h

```diff
@@ -78,6 +78,11 @@ FCITX_CONFIG_ENUM_NAME_WITH_I18N(ShiftLetterKeys,
 
 constexpr char kDefaultOpenFileWith[] = "xdg-open";
 
+enum class CtrlEnterKey { Disabled, InputReading, InputHTMLRubyText };
```

Comment:
Perhaps s/Input/Output/g and s/input_/output_/g here?

Reply:
Fixed.